crates/esthri/src/ops/sync.rs

```diff
@@ -615,6 +650,7 @@ where
                             warn!("s3 etag error: {}", err);
                         }
                     },
+                    Ok(None) => warn!("no local etag"),
```

Comment:
This should only happen if we have a logic error somewhere, right? Would it be more appropriate to panic (or do one of your fancy panic inducing macros? :grin: )

Reply:
Oops missed this one. I switched to a log because we mostly seem to be just logging this kind of errors thought the rest of the code. Like in some places we check to see if an object returned from esthri has an etag, and log/skip it if it doesn't. But really there should never be an object without and etag, it's just a quirk of the client we use